src/components/statevector_component/aggregation.py

```diff
@@ -343,7 +460,15 @@ def force_native_res_pixels(config, clusters, sensitivities):
         cluster_pairs    [(tuple)]: cluster pairings
     Returns:             [double] : updated sensitivities
     """
+    plumes = SRON_plumes(config)
     coords = read_coordinates(config["ForcedNativeResolutionElements"])
+    if plumes is not None:
+        if coords is None:
+            coords = plumes
+        else:
+            coords.extend(plumes)
+        if (len(coords)) > config["NumberOfElements"]:
```

Comment:
I just realized his should only be applied to coordinates with lats and lons that are in different grid cells, since there can be multiple point sources in the same grid cell. I think all you need to do is use the binned lon and binned lats instead (defined lower down), making sure to drop any coords that are the same once binned.